tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/builtinoperations/FreezeAndIsFrozenTest.java

```diff
@@ -597,8 +597,8 @@ public void testDeprecatedWarningForIsReadOnly() {
         CompileResult result = BCompileUtil.compile(
                 "test-src/expressions/builtinoperations/is_readonly_deprecated_warning.bal");
         int index = 0;
-        validateWarning(result, index++, "usage of construct 'x.isReadOnly(x)' is deprecated", 22, 9);
-        validateWarning(result, index++, "usage of construct 'value:isReadOnly(x)' is deprecated", 24, 17);
+        validateWarning(result, index++, "usage of construct 'isReadOnly' is deprecated", 22, 9);
```

Comment:
Can we include more information about the construct here? Like a qualified name instead of just the name?

e.g., `ballerina/lang.value:isReadOnly`

For object methods it may have to be like `Obj.method`.

For anonymous object methods we can probably give only the method name though.

Reply:
Updated